docs/src/examples/components/List/Performance/index.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
+import PerformanceSection from 'docs/src/components/ComponentDoc/PerformanceSection'
+
+const Performance = () => (
+  <PerformanceSection title="Performance">
+    <ComponentExample
+      title="Default"
+      description="A default test."
+      examplePath="components/List/Performance/List.perf"
+    />
```

Comment:
Are the perf results isolated to the `List.perf` and not affected by the use of `ComponentExample`?